plugins/templating/src/main/kotlin/templates/CommandHandler.kt

```diff
@@ -2,10 +2,16 @@ package org.jetbrains.dokka.templates
 
 import org.jetbrains.dokka.base.templating.Command
 import org.jsoup.nodes.Element
+import org.jsoup.nodes.Node
 import java.io.File
 
-interface CommandHandler {
-    fun handleCommand(element: Element, command: Command, input: File, output: File)
+
+interface CommandHandler  {
+    @Deprecated("This was renamed to handleCommandAsTag", ReplaceWith("handleCommandAsTag(command, element, input, output)"))
+    fun handleCommand(element: Element, command: Command, input: File, output: File) =
+        handleCommandAsTag(command, element, input, output)
+    fun handleCommandAsTag(command: Command, body: Element, input: File, output: File)
+    fun handleCommandAsComment(command: Command, body: List<Node>, input: File, output: File) { }
```

Comment:
I think with these changes external plugins that override `CommandHandler` might be broken 

External plugins currently override `handleCommand`, but we only call `handleCommandAsTag` from Dokka. So nothing will happen for the plugins other than compilation error for not overriding `handleCommandAsTag`

Looks like we should call legacy `handleCommand` from `handleCommandAsTag` to give people time to migrate
```suggestion
    fun handleCommand(element: Element, command: Command, input: File, output: File) { }
    fun handleCommandAsTag(command: Command, body: Element, input: File, output: File) = handleCommand(body, command, in, out)
    fun handleCommandAsComment(command: Command, body: List<Node>, input: File, output: File) { }
```

Reply:
So we will have no way to migrate to `handleCommandAsTag` if `handleCommand` is called.
In this case a user implementation of new interface  should have 
``` kt
    fun handleCommand(element: Element, command: Command, input: File, output: File) =
        handleCommandAsTag(command, element, input, output)
```